color_palette/styles.css

```diff
@@ -0,0 +1,80 @@
+:root {
+    --init: black;
+}
+
+.container-1 {
+    display: flex;
+    flex-direction: row;
+}
+
+
+.main {
+    width: 10rem;
+    height: 20rem;
+    margin: .5rem;
+    text-align: center;
+}
+
+.box-1 {
+    background-color: var(--init);
+}
+
+.box-2 {
+    background-color: var(--init);
+}
+
+.box-3 {
+    background-color: var(--init);
+}
```

Comment:
Why did you repeat this class `.box-*` multiple times?

All elements can have the same class `.box-1`, it will work :)

Reply:
Not sure to be honestly at this point...xD You definitely right